src/haxeLanguageServer/features/DiagnosticsFeature.hx

```diff
@@ -2,21 +2,26 @@ package haxeLanguageServer.features;
 
 import haxe.Json;
 import haxe.io.Path;
-import haxeLanguageServer.helper.TypeHelper;
 import haxeLanguageServer.Configuration;
-import haxeLanguageServer.helper.PathHelper;
+import haxeLanguageServer.LanguageServerMethods;
+import haxeLanguageServer.helper.DocHelper;
 import haxeLanguageServer.helper.ImportHelper;
-import haxeLanguageServer.server.DisplayResult;
+import haxeLanguageServer.helper.PathHelper;
+import haxeLanguageServer.helper.TypeHelper;
 import haxeLanguageServer.helper.WorkspaceEditHelper;
-import haxeLanguageServer.LanguageServerMethods;
+import haxeLanguageServer.server.DisplayResult;
 import js.node.ChildProcess;
 
 using Lambda;
 
 class DiagnosticsFeature {
 	static inline var DiagnosticsSource = "diagnostics";
 	static inline var RemoveUnusedImportUsingTitle = "Remove unused import/using";
+	#if debug
+	static inline var RemoveAllUnusedImportsUsingsTitle = "Organize imports/usings";
+	#else
 	static inline var RemoveAllUnusedImportsUsingsTitle = "Remove all unused imports/usings";
```

Comment:
We still want to have this available as a separate code action, just as a `QuickFix` instead of `SourceOrganizeImports`.